packages/fxa-payments-server/src/components/PaymentForm/index.scss

```diff
@@ -52,7 +52,9 @@ form.payment {
   }
 
   .terms {
+    display: flex;
     font-size: 15px;
+    justify-content: center;
     padding-top: 56px;
```

Comment:
Clicking “Terms of service” clickable area is very large due to using `padding`. You can use `margin` instead, and make sure it’s not clickable to the left/right of the link (it probably needs to be inline or inline-flex; you may need a span or a container)